README.md

```diff
@@ -81,14 +81,49 @@ If you are using Atom, make sure to install the [`prettier-atom`](https://atom.i
 All the configuration for prettier and xo is in the `package.json`.
 You shouldn't have to configure things separately, please file a issue if there's a problem.
 
-### 📖 Step 2. Get storybook up and running
+### 🍴 Step 2. Fork this repository
+
+In order to contribute to Evergreen, you need to fork this repo, and develop on your own local clone.
+
+If you don't know how to do so, follow [this](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) guide
+
+### 📖 Step 3. Get storybook up and running
+
+First, `cd` into your local cloned repository and install your `node_modules` with:
+
+```
+$ yarn
+```
 
 To actually start seeing the components you have to run React Storybook:
 
 ```
 $ yarn dev
 ```
 
+### 🏆 Step 4. Making your pull request
+
+Once you're done with making your changes, push everything to your local repository's branch.
+
+From here, you can open up a pull request from your forked repository's branch into `segmentio/evergreen`'s `master` branch.
+
+In your PR description, explain the changes you made, why you made them, how to test them, and anything that might be a point of interest.
+
+Once you create your PR, it will be reviewed and hopefully merged quickly!
+
+### 📓 Step 5. Run evergreen docs locally
```

Comment:
I think we should can encourage folks to update any relevant docs and storybook stories for changes or additions they've made. Perhaps in "Step 4. Making your pull request"